src/surfing_penguin/routes.py

```diff
@@ -18,6 +19,16 @@
         'id': 1
     }
 ]
```

Comment:
For testing, you should add some unit tests. See https://docs.pytest.org/en/latest/